technical_onboarding/our_awesome_list.md

```diff
@@ -0,0 +1,34 @@
+# Our Awesome List
+
+### Python
+* [Python - Udemy - Jose Portilla](https://www.udemy.com/complete-python-bootcamp/)
+
+
+### Django
+* [Django Girls](https://tutorial.djangogirls.org/pl)
+
+### Flask
+* [Rest APIs with Flask and Python - Udemy - Jose Salvatierra](https://www.udemy.com/rest-api-flask-and-python)
+
+
+### Docker
+* [Post Installation Steps For Linux](https://docs.docker.com/install/linux/linux-postinstall)
+
+
+### Git
+* [Git flight rules](https://github.com/k88hudson/git-flight-rules/blob/master/README.md)
+* [Learn Git - Codecademy](https://www.codecademy.com/learn/learn-git)
+
+### Linux
+
```

Comment:
Danuta wrzuciła jeszcze to w #51 http://ubuntuhandbook.org/index.php/2016/05/time-differences-ubuntu-1604-windows-10/

To bardzo ważny link!